Training_Center_Task_11_Selenium_Tests/ConstantsTests.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Training_Center_Task_11_Selenium_Tests
+{
+    public class ConstantsTests
+    {
+        public const int WebDriverWaitTime = 10;
+        public const string WebDriverPath = @"C:\Users\IvoIgov\source\repos\Training_Center_Task_11\Training_Center_Task_11_Selenium_Tests\bin\Debug\net6.0\";
+
+        public const string ScreenshotsFilePath = @"C:\Users\IvoIgov\source\repos\Training_Center_Task_11\Screenshots";
```

Comment:
please do not use hardcoded path.

Reply:
Fixed hardcoded paths for chromedriver, screenshots, JSON. Removed classes with Constants